features/lorawan/system/LoRaWANTimer.cpp

```diff
@@ -0,0 +1,84 @@
+/**
+ / _____)             _              | |
+( (____  _____ ____ _| |_ _____  ____| |__
+ \____ \| ___ |    (_   _) ___ |/ ___)  _ \
+ _____) ) ____| | | || |_| ____( (___| | | |
+(______/|_____)_|_|_| \__)_____)\____)_| |_|
+    (C)2013 Semtech
+
+Description: Timer objects and scheduling management
+
+License: Revised BSD License, see LICENSE.TXT file include in the project
+
+Maintainer: Miguel Luis and Gregory Cristian
+
+
+Copyright (c) 2017, Arm Limited and affiliates.
+
+SPDX-License-Identifier: BSD-3-Clause
+*/
+
+#include "lorawan/system/LoRaWANTimer.h"
+
+static Timer TimeCounter;
+static Ticker LoadTimeCounter;
+
+volatile uint32_t CurrentTime = 0;
```

Comment:
Please review `CurrentTime ` -> `current_time` , plus others in this file. same is for the name of this file, it acts like C++ class API, I would be looking here for `LoraWanTimer C++ object` , but would find here TimerCounter

Reply:
I understand your point. And we are aware of the style blasphemies we are making :) The reason is that we don't want to change any APIs or variables that are being used in LoRaMac.cpp